docs/build/isc/v1.1/docs/how-tos/core-contracts/nft/mint-nft.md

```diff
@@ -22,9 +22,16 @@ The Stardust update allows you to create your own NFTs. You can also use [IRC27]
 ```solidity
 ISCAgentID memory agentID = ISC.sandbox.getSenderAccount();
 ```
+:::info ISC AGENT ID 
+The `ISCAgentID` represents the identifier of the agent (user or contract) whose NFTs you want to retrieve. You can get the `AgentID` from the sender by calling `ISC.sandbox.getSenderAccount()`.
+:::
 
 3. Create an `IRC27Metadata` struct  with all the needed data:
```

Comment:
Could you turn these into headers and add a small explanation for the step? This way, we can link to them. 

### 3. Create an `IRC27Metadata` Struct 

Next, you should create the [`IRC27NFTMetadata`](/tips/tips/TIP-0027/). This struct will be used to...  This example will use the `_uri`, `_mimeType`, and `_name` param from the contract. You can also set the `collectionName`, `royalties`, [highlights of the list on /tips/tips/TIP-0027/#nft-schema].